prow/config/jobs/ztunnel.yaml

```diff
@@ -14,6 +14,11 @@ jobs:
     types: [presubmit]
     requirements: [cratescache]
 
+  - name: coverage
+    command: [entrypoint, make, coverage]
+    types: [postsubmit]
+    requirements: [cratescache]
```

Comment:
Will this break the cache since `make converage` will compile with some extra RUSTC flags?

Reply:
I don't know enough about prow or `cratescache` to know.

Unless `cratescache` is doing something very strange, it should cache pulled crates individually - so if we have 80% precached from previous jobs and `coverage` pulls a net new 20%, we should be fine.

Reply:
Oh it's just a persistently mounted `.cargo` dir, and not something with any build intelligence.

Meh, I am inclined to not care - this is a postsubmit job anyway. If we want to make it a presubmit job, execution time matters more, but we aren't doing that.

Reply:
@Stevenjin8 if you're feeling like it, you can create a dedicated `cratescoveragecache` mount and use that here instead. That will use considerably more disk space tho.

Reply:
Yeah its not a build cache. Would love to have one but it doesn't work well with cargo since the builds take an exclusive lock on the cache..